generator/operation_test.go

```diff
@@ -27,35 +27,11 @@ import (
 func TestUniqueOperationNames(t *testing.T) {
 	doc, err := spec.Load("../fixtures/codegen/todolist.simple.yml")
 	if assert.NoError(t, err) {
-		sp := doc.Spec()
-		sp.Paths.Paths["/tasks"].Post.ID = "saveTask"
-		sp.Paths.Paths["/tasks"].Post.AddExtension("origName", "createTask")
-		sp.Paths.Paths["/tasks/{id}"].Put.ID = "saveTask"
-		sp.Paths.Paths["/tasks/{id}"].Put.AddExtension("origName", "updateTask")
-
-		ops := gatherOperations(doc, nil)
-		assert.Len(t, ops, 4)
-		_, exists := ops["saveTask"]
-		assert.True(t, exists)
-		_, exists = ops["saveTask1"]
-		assert.True(t, exists)
-	}
-}
-
-func TestEmptyOperationNames(t *testing.T) {
-	doc, err := spec.Load("../fixtures/codegen/todolist.simple.yml")
-	if assert.NoError(t, err) {
-		sp := doc.Spec()
-		sp.Paths.Paths["/tasks"].Post.ID = ""
-		sp.Paths.Paths["/tasks"].Post.AddExtension("origName", "createTask")
-		sp.Paths.Paths["/tasks/{id}"].Put.ID = ""
-		sp.Paths.Paths["/tasks/{id}"].Put.AddExtension("origName", "updateTask")
-
```

Comment:
where did those tests go? I can't seem to find tests that continue to test for this? Are they no longer required?